apps/configurator/lib/configurator_web/controllers/game_configuration_html/show.html.heex

```diff
@@ -21,7 +21,8 @@
   <:item title="Zone stop interval ms"><%= @game_configuration.zone_stop_interval_ms %></:item>
   <:item title="Zone start interval ms"><%= @game_configuration.zone_start_interval_ms %></:item>
   <:item title="Zone damage interval ms"><%= @game_configuration.zone_damage_interval_ms %></:item>
-  <:item title="Zone damage"><%= @game_configuration.zone_damage %></:item>
```

Comment:
We lost this

Reply:
Whoops

Reply:
Done in [fix: return zone damage](https://github.com/lambdaclass/mirra_backend/pull/849/commits/18eee02271a9e7c6ebace70c57f3517fcaa512f2)